dm/dm/master/server.go

```diff
@@ -2147,9 +2148,57 @@ func (s *Server) GetCfg(ctx context.Context, req *pb.GetCfgRequest) (*pb.GetCfgR
 	if shouldRet {
 		return resp2, err2
 	}
-	// For the get-config command, you want to filter out fields that are not easily readable by humans,
-	// such as SSLXXBytes field in `Security` struct
+
+	formartTaskString := func(subCfgList []*config.SubTaskConfig) string {
+		sort.Slice(subCfgList, func(i, j int) bool {
```

Comment:
we can move this sort inside `SubTaskConfigsToTaskConfig`

Reply:
yes ,but we still need sort for get task config, so left sort part in this func